frontend/public/components/edit-yaml.jsx

```diff
@@ -196,7 +202,8 @@ export const EditYAML = connect(stateToProps)(
         return;
       }
 
-      if (!this.props.model) {
+      const model = this.getModel(obj);
+      if (!model) {
         this.handleError(`"${obj.kind}" is not a valid kind.`);
```

Comment:
This would be clearer as:
```javascript
this.handleError(`The server doesn't have a resource type "kind: ${obj.kind}, apiVersion: ${obj.apiVersion}".`);
```